BLUEPRINTS.md

```diff
@@ -43,6 +43,6 @@ For this reason try to _avoid exact version at blueprint dependencies_ and disab
 - priorities names are static constants at the generator class.
   `get initializing() {}` -> `get [Generator.INITIALIZING]() {}`
   Motivation can be found at https://github.com/jhipster/generator-jhipster/blob/main/rfcs/3-jhipster-rfc-unambiguous-priorities.md.
-- many property migration can be warned by enabling `jhipster7Migration` feature at the constructor.
+- much property migration can be warned by enabling `jhipster7Migration` feature at the constructor.
```

Comment:
much looks weird to me, not a native speaker.

Reply:
I think "many" sounds better, but I'm also trying to understand what the sentence should say. Are you trying to say many properties can be migrated automatically, or the deprecated ones can be printed to the console?

Reply:
Removed properties will work but printed to console.
Won't work with the flag disabled.